### PR TITLE
German Translation

### LINKED ISSUE #1: Translate Description
- [x] Translate Short Description
- [x] Translate Description

CheckIO-to-German/Roadmap#84

### PR DESCRIPTION
**Pull request by DahliaSR**
Closes #1

### CODE REVIEW

**Comment by DahliaSR** [comment]
@GiForce 
Your approval is pending...

File: translations/de/info/task_description.html [hunk 1]
```diff
@@ -129,41 +135,43 @@
...
+        And burbled as it came!""", "noir") == [4, 16, 7, 16]
     </pre>
 </div>
 
 <p class="for_info_only">
-    <strong>How it is used: </strong>
-    This task shows the variance of the positional ciphers.
-    With this cipher you can hide a message within a book which allows you and receiver to send these coded messages.
+    <strong>Wozu das gut ist: </strong>
+    Diese Aufgabe zeigt dir eine Variante der <a href="https://de.wikipedia.org/wiki/Buch-Verschl%C3%BCsselung">Buch- oder auch Ottendorf-Verschlüsselung</a>.
```
Comment (GiForce) [inline translations/de/info/task_description.html]: ...zeigt **D**ir...

Comment (DahliaSR) [reply translations/de/info/task_description.html]: :white_check_mark:

**Review by GiForce — APPROVED** [review]
🎃

**Review by DarkoRevan — APPROVED** [review]

**Review by DarkoRevan — APPROVED** [review]

**Pull request merged**